src/types/config.js

```diff
@@ -12,9 +12,10 @@
  * @prop {string} authority
  * @prop {string} grantToken
  * @prop {string} [icon]
- * @prop {string[]|Promise<string[]>} [groups] -
- *   List of groups to show. The embedder can specify an array. In the sidebar
- *   this may be converted to a Promise if this information is fetched asynchronously.
+ * @prop {string[]|Promise<string[]>|'$rpc:requestGroups'} [groups] -
```

Comment:
Yep, this was a change I was proposing in some adjacent work that is taking a timeout (i.e. "great minds").